docs/develop-resources/api-conventions/validations.md

```diff
@@ -57,8 +57,18 @@ type CloudBuildWorkerPoolSpec struct {
 
 Validates the immutability in the controller runtime, by comparing the difference between `spec` and `status`, together with the corresponding proto field "IMMUTABLE" **field_behavior**. (default behavior if using the direct auto-generated code)
 
-To see if you can use this approach, check if the resource proto supports **field_behavior** (See [AIP 203](https://google.aip.dev/203)). 
-If so, you just need to make sure
+All google services are located in [googleapis Github repo](https://github.com/googleapis/googleapis/tree/master/google),
+refer to your resource's API documentation to identify the service name, for example: [compute](https://cloud.google.com/compute/docs/reference/rest/v1).
+Once you identify the service, find the proper path to the proto files, for example:
+[`google/cloud/compute/v1/compute.proto`](https://github.com/googleapis/googleapis/blob/master/google/cloud/compute/v1/compute.proto).
+To see if you can use this approach, check if the resource proto supports **field_behavior** (See [AIP 203](https://google.aip.dev/203)).
+
+Search for the resource name, like `message TargetTcpProxy`, this resource does not support immutable field_behavior.
+SecureSourceManagerInstance resource, for example `message Instance` in [secure_source_manager.proto](https://github.com/googleapis/googleapis/blob/master/google/cloud/securesourcemanager/v1/secure_source_manager.proto),
+field [`kms_key`](https://github.com/googleapis/googleapis/blob/master/google/cloud/securesourcemanager/v1/secure_source_manager.proto#L398)(KCC type KmsKeyRef) 
```

Comment:
Can we use a non reference field example? That seems to be more common.

Reply:
I was not able to find another example in our code base... not all resources/fields we currently support, have field_behavior. I feel option 2 is more common. 

I can update the example when I find a non-reference field. But here I just want to give contributor an idea  what's the `field_behavior` look like in proto file.